### PR TITLE
Configure monasca-server in keystone

### PR DESCRIPTION
**Pull request by samirjorina**

### CODE REVIEW

File: chef/cookbooks/monasca/recipes/server.rb [hunk 1]
```diff
@@ -30,3 +30,80 @@
...
+
+my_admin_host = CrowbarHelper.get_host_for_admin_url(node, ha_enabled)
+my_public_host = CrowbarHelper.get_host_for_public_url(
+  node, ssl_enabled, ha_enabled
+)
+
+register_auth_hash = { user: keystone_settings["admin_user"],
+                       password: keystone_settings["admin_password"],
+                       tenant: keystone_settings["admin_tenant"] }
+
```
Comment (matrixik) [inline chef/cookbooks/monasca/recipes/server.rb]: Too many empty lines

File: chef/cookbooks/monasca/recipes/server.rb [hunk 2]
```diff
@@ -30,3 +30,80 @@
...
+keystone_register "monasca server wakeup keystone" do
+  protocol keystone_settings["protocol"]
+  insecure keystone_settings["insecure"]
+  host keystone_settings["internal_url_host"]
+  port keystone_settings["admin_port"]
+  auth register_auth_hash
+  action :wakeup
+end
+
+
```
Comment (matrixik) [inline chef/cookbooks/monasca/recipes/server.rb]: Too many empty lines

File: chef/cookbooks/monasca/recipes/server.rb [hunk 3]
```diff
@@ -30,3 +30,80 @@
...
+ssl_enabled = node[:monasca][:api][:ssl]
+monasca_protocol = ssl_enabled ? "https" : "http"
+ha_enabled = node[:monasca][:ha][:enabled]
+
+my_admin_host = CrowbarHelper.get_host_for_admin_url(node, ha_enabled)
+my_public_host = CrowbarHelper.get_host_for_public_url(
+  node, ssl_enabled, ha_enabled
+)
+
+register_auth_hash = { user: keystone_settings["admin_user"],
```
Comment (matrixik) [inline chef/cookbooks/monasca/recipes/server.rb]: Move `user:...` to next line and align.

**Review by matrixik — CHANGES_REQUESTED** [review]

File: chef/cookbooks/monasca/libraries/helpers.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
+module MonascaHelper
```
Comment (matrixik) [inline chef/cookbooks/monasca/libraries/helpers.rb:1]: Copyright

**Review by matrixik — CHANGES_REQUESTED** [review]

File: chef/cookbooks/monasca/recipes/role_monasca_server.rb [hunk 5]
```diff
@@ -16,5 +16,6 @@
 
 if CrowbarRoleRecipe.node_state_valid_for_role?(node, "monasca", "monasca-server")
   include_recipe "monasca::server"
-#  include_recipe "monasca::api"
+  include_recipe "monasca::api"
```
Comment (kornicameister) [inline chef/cookbooks/monasca/recipes/role_monasca_server.rb:19]: If we plan on keeping it that way, I think that should go before monasca::server if the purpose of those recipes is just to register stuff in keystone.

File: chef/cookbooks/monasca/libraries/helpers.rb [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    CrowbarHelper.get_host_for_admin_url(node, ha_enabled)
+  end
+
+  def self.api_admin_url(node)
+    host = api_admin_host(node)
+    protocol = node[:monasca][:api][:ssl] ? "https" : "http"
+    port = node[:monasca][:api][:bind_port]
+    "#{protocol}://#{host}:#{port}/v2.0"
+  end
+end
```
Comment (kornicameister) [inline chef/cookbooks/monasca/libraries/helpers.rb:26]: What about log-api ?

**Pull request closed without merging**